src/bson.ts

```diff
@@ -31,8 +30,9 @@ export type { ObjectIdExtended, ObjectIdLike } from './objectid';
 export type { BSONRegExpExtended, BSONRegExpExtendedLegacy } from './regexp';
 export type { BSONSymbolExtended } from './symbol';
 export type { LongWithoutOverrides, TimestampExtended, TimestampOverrides } from './timestamp';
-export { LongWithoutOverridesClass } from './timestamp';
+export type { LongWithoutOverridesClass } from './timestamp';
```

Comment:
This is typescript hackery to support Timestamp subclassing Long, this isn't a real value that should be exported, removing this change came about as part of cleaning up the bundle output